test/extended/networking/acl_audit_log.go

```diff
@@ -0,0 +1,213 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	networkingv1 "k8s.io/api/networking/v1"
+
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[sig-network] acl-logging", func() {
+	oc := exutil.NewCLI("acl-logging")
+
+	// The OVNKubernetes subnet plugin should allow acl_logging for network policy.
+	// For third party plugins, the behavior is unspecified and we should not run either test.
+	InPluginContext([]string{},
```

Comment:
`InPluginContext` is a hack for running under some openshift-sdn Modes but not others. It's not what you want here, though you might want to add a new `InOVNKubernetesContext` that works similarly but checks the `network.config.openshift.io` object rather than the old sdn `ClusterNetwork` object

Reply:
Ack that makes much more sense, thanks!